.gitignore

```diff
@@ -7,4 +7,5 @@ venv/
 .pytest_cache/
 **__pycache__/
 *.pyc
-app/db.sqlite3
\ No newline at end of file
+app/db.sqlite3
+**.sqlite3
```

Comment:
no blank line at the end of file

Reply:
sorre, I'll be more careful next time. thanks for aproved